docs/install.rst

```diff
@@ -279,6 +281,69 @@ If you have already activated the correct environment, but you still do not see
 Frequently asked questions
 --------------------------
 
+**I tried the installation instructions, but I cannot import Pylake inside a Jupyter notebook**
+
+If Python claims there is no package with the name `lumicks.pylake` or it fails with a `DLL load failed` error, even after you're followed the above installation instructions, then please check the following:
+
+* You should be launching the Jupyter Notebook from within the same environment where you installed `pylake`.
+
+* If you already had an installation of Anaconda, and you installed `pylake` into a new environment alongside existing environments, it may be that you have a conflict between multiple Jupyter installations.
+
+In this case, try running the following command from an Anaconda Prompt::
+
+    jupyter kernelspec list
+
+This lists all the kernels that have been installed (thereby making them available to Jupyter).
+
+If the output of this command lists any paths containing `AppData\\Roaming\\jupyter\\kernels` or `C:\\ProgramData\\jupyter\\kernels` (Windows); `.local/share/jupyter/kernels` or `/usr/share/jupyter/kernels` or `/usr/local/share/jupyter/kernels` (Mac/Linux); or `/Library/Jupyter/kernels` (Mac); then invoke::
+
+    python -m ipykernel install --user --name=envname
+
+Where `envname` should be replaced with the name of the environment you are using.
+
+Now restart the Jupyter Notebook, and make sure you open your Notebook using the `pylake` kernel that's now available in the list.
+
+*Why does this work?*
+
+Jupyter notebooks work by sending code that you run in a notebook to a separate "computational engine" which executes this code.
+This computational engine is called a kernel.
+To do this, Jupyter has to first identify which kernel to use.
+Jupyter searches for kernels based on a list specified in the `kernelspec`.
+
+When using Anaconda, Jupyter will by default use the Python kernel that corresponds to that environment.
+By default, this `kernelspec` will be called `python3`.
+However, if an explicit `kernelspec` with that name has been created in the all-users or per-user kernel registry, Jupyter no longer performs auto-detection of the IPython kernel in the current conda environment.
+That means you are no longer able to start a Jupyter kernel from the currently active environment without explicitly installing it.
+
+*Can I just revert back to detecting which kernel to use based on the environment instead?*
+
+Yes. Alternatively, one can remove the kernelspec that is causing the issue, resulting in `Anaconda` reverting back to the default behaviour of using the kernel for the active environment::
+
+    jupyter kernelspec uninstall envname
+
+Where `envname` needs to be the name of the registered kernel that is being loaded. Usually this will be `python3`.
```

Comment:
Very clear! I finally start to understand what went wrong here.

Reply:
:)